src/test/java/teammates/test/cases/storage/FeedbackSessionsDbTest.java

```diff
@@ -319,12 +319,18 @@ public void testUpdateFeedbackSession() throws Exception {
     }
 
     private FeedbackSessionAttributes getNewFeedbackSession() {
+        Instant sessionStartTime = Instant.now();
+        try {
+            Thread.sleep(1);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
         return FeedbackSessionAttributes.builder("fsTest1", "testCourse", "valid@email.com")
-                .withCreatedTime(Instant.now())
-                .withStartTime(Instant.now())
+                .withCreatedTime(sessionStartTime)
+                .withStartTime(sessionStartTime)
```

Comment:
I don't think `Thread.sleep(1)` will help here.
This will ensure that `createdTime`,`withStartTime`, `SessionVisibleFromTime`, `ResultsVisibleFromTime` have same instant which is opposite, to what is proposed, to be solved by this PR.